WcaOnRails/spec/controllers/teams_controller_spec.rb

```diff
@@ -161,5 +161,17 @@
         expect(team.reload.team_members.first.user.id).to eq member.id
       end
     end
+
+    context "Senior Delegates managing the probation team, despite not being a member of the probation team" do
```

Comment:
Could we also add tests for the WFC Leader and a WFC Senior Member?

Besides, I think we never capitalize stuff here.

Reply:
We should probably add tests like [this one](https://github.com/thewca/worldcubeassociation.org/pull/4144/files#diff-dadb4202f6cdc5dc7b128619017dc973R63) too.

Reply:
sweet for the capitalization. It seems we do capitalize stuff like WDC, so Ill do the same for WFC and then leave the other words lowercase.